src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -1716,6 +1716,8 @@ CLParser::ParseCommandLine(int argc, char** argv)
   // Overriding the max_threads default for request_rate search
   if (!params_->max_threads_specified && params_->targeting_concurrency()) {
     params_->max_threads = 16;
+    params_->max_threads =
```

Comment:
Could you merge this line with the line right before it? It seems like a redundant assignment if you're just getting the max of 16 and concurrency_range's last value. Also, is concurrency range always ascending/sorted, or do you need to get the max value in the range here?

Reply:
AFAIK, concurrency range is ascending according to the documentation [here](https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/docs/cli.md#:~:text=Specifies%20the%20range%20of%20concurrency%20levels%20covered%20by%20Perf%20Analyzer.%20Perf%20Analyzer%20will%20start%20from%20the%20concurrency%20level%20of%20%27start%27%20and%20go%20until%20%27end%27%20with%20a%20stride%20of%20%27step%27.)

Reply:
Yea. It is always ascending.
Agree on the merging of the two lines.
Ideally, we leverage a default constant somewhere that holds the 16 but is more descriptive than a magic number.